doc/nrf/libraries/emds/emds.rst

```diff
@@ -0,0 +1,50 @@
+.. _emds_readme:
+
+Emergency data storage
+######################
+
+.. contents::
+    :local:
+    :depth: 2
+
+The emergency data storage (EMDS) library provides functionality that will prevent wear and tear of flash as data that are changing often are stored in memory, but can be stored in flash and be restored after reboot. It will store data within a guaranteed time, based on the amount of data that are being stored.
+EMDS can be used in situations of critical power shortage to store data in memory, e.g. before the battery that powers a device is depleted. It is important that the HW has appropriate functionality to sustain power long enough for storage to be completed before the device is fully discharged.
```

Comment:
```suggestion
The emergency data storage (EMDS) library provides persistent storage functionality that is designed to prevent wear and tear of flash memory. It is intended for storing data that undergoes frequent changes during runtime. To prevent frequent writes to the flash memory, the EMDS library will only write data to flash at the point when the device is shuting down. At reboot the module will restore this data. EMDS can store data within a guaranteed time, based on the amount of data that are being stored.
EMDS can be used in situations of critical power shortage to store data in memory, e.g. before the battery that powers a device is depleted. It is important that the HW has appropriate functionality to sustain power long enough for storage to be completed before the device is fully discharged.
```